Kernel_23/include/CGAL/Weighted_point_3.h

```diff
@@ -91,62 +91,62 @@ class Weighted_point_3 : public R_::Kernel_base::Weighted_point_3
     : Rep(typename R::Construct_weighted_point_3()(Return_base_tag(), x, y, z))
   {}
 
-  typename cpp11::result_of<typename R::Construct_point_3( Weighted_point_3)>::type
+  typename cpp11::result_of<typename R::Construct_point_3(Weighted_point_3)>::type
   point() const
   {
     return typename R::Construct_point_3()(*this);
   }
 
-  typename cpp11::result_of<typename R::Compute_weight_3( Weighted_point_3)>::type
+  typename cpp11::result_of<typename R::Compute_weight_3(Weighted_point_3)>::type
   weight() const
   {
     return typename R::Compute_weight_3()(*this);
   }
 
 
-  typename cpp11::result_of<typename R::Compute_x_3( Weighted_point_3)>::type
+  typename cpp11::result_of<typename R::Compute_x_3(Point_3)>::type
```

Comment:
Andreas, after this change, then maybe the `std::result_of` can be used instead of TR1.